drizzle-kit/src/cli/connections.ts

```diff
@@ -681,13 +681,19 @@ export const connectToSQLite = async (
 				params,
 				method,
 			) => {
+				let body:string;
```

Comment:
prettier would format this as:
```suggestion
				let body: string;
```

Reply:
thank you, fixed